src/styles/utilities.scss

```diff
@@ -44,3 +44,57 @@
 .selected-box-shadow {
   box-shadow: 2px 0px 10px 0px var(--boxShadow-primary)
 }
+
+/* FONT SIZES */
+.text-xs {	
```

Comment:
question: Could we do something similar to: https://github.com/jbx-protocol/juice-interface/blob/main/src/hooks/JuiceTheme.ts#L8 and https://github.com/jbx-protocol/juice-interface/blob/main/src/models/semantic-theme/colors.ts#L22



The goal would be to get these constants in but also get some type checking in like colors.